### DISCUSSION TITLE
Q: NGINX version

### DISCUSSION THREAD

**Discussion by DaleyKD** [nginx-proxy/nginx-proxy#1656] [General]
_I tried to go to the Google Groups forum, but it doesn't seem to be available anymore?_

I'm just curious to know what the general idea is to keep NGINX updated?  It looks like they just released 1.19.10 and we're on 1.19.3.  It's not a big deal, but I didn't know if maybe it was planned to update quarterly?  Yearly?  Ever?

Thanks for all of your hard work.  This image has made local dev glorious.

**Comment by buchdag** [comment]
> I tried to go to the Google Groups forum, but it doesn't seem to be available anymore?

Nope, it's been dead for a while, the URL has been removed for now.

> I'm just curious to know what the general idea is to keep NGINX updated? It looks like they just released 1.19.10 and we're on 1.19.3. It's not a big deal, but I didn't know if maybe it was planned to update quarterly? Yearly? Ever?

The general idea is to keep it up to date with the latest semver `mainline` release of nginx, but updates are delayed until we have a working CI again. Please see https://github.com/nginx-proxy/nginx-proxy/issues/1304#issuecomment-800318195 and the following comments for more informations.

**Reply by buchdag** [reply]
@DaleyKD I'm reviving this issue as a discussion: do you think it would make sense / be helpful to users of this image to have two additional flavours of the image based upon `nginx:stable` (and `nginx:stable-alpine`) instead of just sticking to `nginx:mainline` ?

**Reply by DaleyKD** [reply]
MAAAAAAYBE?

I honestly don't keep up with NGINX much.  My main use-case is to keep close to the version of k8s/ingress-nginx which is used by our AKS cluster.  (https://github.com/kubernetes/ingress-nginx#support-versions-table)

I use this KILLER proxy (thanks again!) to help us with all our microservices in local dev and the docker-compose files for each of the services.

All of that to explain that I don't have a particular care for `stable` vs. `mainline`, except for the above reason.  :smile:

**Comment by DaleyKD** [comment]
Thank you so much!  Exactly what I wanted to know.  Good luck on getting CI going.

**Comment by buchdag** [comment]
@DaleyKD updated nginx to `1.19.10` in #1590

**Comment by DaleyKD** [comment]
@buchdag : Thank you so much.  This is exciting to see this project get the attention it deserves.

Not that it's a big deal, but the docker page doesn't seem to be fully updated (and the profile picture is broken):
![image](https://user-images.githubusercontent.com/7206282/116253159-d85cd580-a735-11eb-9009-680a7ca9d44c.png)

**Comment by buchdag** [comment]
The org does not have any picture on DockerHub (or on GitHub for that matter) and it looks like it's not playing nicely with DockerHub. Unless someone is super motivated to create a cool logo, it looks like it'll stay this way 😅 

The DockerHub readme isn't updated automatically and the nginx badge is a static svg, so I have to update it on both GitHub and DockerHub every time the nginx version is updated, yay 😒
